test/functional/repository/remove_content_test.rb

```diff
@@ -104,5 +104,21 @@ module HammerCLIKatello
                    --ids 20,21,22))
       end
     end
+
+    it 'does not resolves repository id without product details' do
+      api_expects_no_call
+      result = run_cmd(%w(repository remove-content --ids 1 --name repo1))
+      assert_includes(result.err, 'At least one of options --product, --product-id is required')
+    end
+
+    it 'does not resolves repository id without organization details' do
```

Comment:
This test already exists: https://github.com/Katello/hammer-cli-katello/blob/master/test/functional/repository/remove_content_test.rb#L39